helm/chart/router/values.yaml

```diff
@@ -7,7 +7,7 @@ replicaCount: 1
 # -- See https://www.apollographql.com/docs/router/configuration/overview#configuration-file for yaml structure
 router:
   configuration:
-    graphql:
+    supergraph:
```

Comment:
this was not a part of this PR intentionally, but is the result of me getting the changes from #1718 after running:

```
helm template --set router.configuration.telemetry.metrics.prometheus.enabled=true  --set managedFederation.apiKey="REDACTED" --set managedFederation.graphRef="REDACTED" --debug .
```

.. I think.